packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -294,6 +294,8 @@
     "@contribute_develop_text": {},
     "contribute_develop_text_2": "You can join the Open Food Facts Slack chatroom which is the preferred way to ask questions.",
     "@contribute_develop_text_2": {},
+    "contribute_develop_dev_mode_title": "DEV Mode?",
```

Comment:
Could you remove the old dev mode translations

Reply:
OK, it's done